src/app/src/components/Submissions/Badges.js

```diff
@@ -26,9 +10,12 @@ const BADGE_COLORS = {
     [BOUNDARY_STATUS.APPROVED]: 'green',
 };
 
-export function StatusBadge({ status }) {
+export function StatusBadge({ status, fixedHeight }) {
     return (
-        <Badge variant='solid' colorScheme={BADGE_COLORS[status]}>
+        <Badge
+            variant={fixedHeight ? 'solidFixedHeight' : 'solid'}
```

Comment:
Let's just use `solid` here and drop the `solidFixedHeight`. `solid` is enough to make it match the wireframes.

Reply:
eed95eb